staging/src/k8s.io/apimachinery/pkg/util/net/util.go

```diff
@@ -38,25 +38,22 @@ func IPNetEqual(ipnet1, ipnet2 *net.IPNet) bool {
 	return false
 }
 
-// Returns if the given err is "connection reset by peer" error.
-func IsConnectionReset(err error) bool {
+// Returns true if the given err is "connection refused" error.
+func IsConnectionRefused(err error) bool {
```

Comment:
@mfojtik in your rebase, these got reordered.  Check to see if this change is even needed anymore.

Reply:
this is weird, but I see both functions (reset and refused) are present in file and needed, not sure why this was re-rdered.